app/controllers/users.js

```diff
@@ -0,0 +1,24 @@
+const errors = require('../errors');
+const logger = require('../logger');
+const usersService = require('../services/users');
+
+const signUp = async (req, res, next) => {
+  const userData = req.body;
+  try {
+    const userFound = await usersService.findByEmail(userData.email);
+    if (userFound) {
+      logger.error('The email provided is already linked to an account');
+      return next(errors.conflict('The email provided is already linked to an account'));
```

Comment:
Add this error messages to constants, then in the unit test you only have to import it.